spec/features/admin/view_unconfirmed_users_spec.rb

```diff
@@ -24,5 +25,9 @@
       expect(page).to have_content(unconfirmed_user_2.email)
       expect(page).to_not have_content(confirmed_user.email)
     end
+
+    it 'displays the list of all unconfirmed users in alphabetical order' do
+      expect(page.body).to match(/test2@gov.uk.*test3@gov.uk/m)
```

Comment:
I'm unsure on how much this test proves.

It's possible for the code to be ordering it based on when a row was created.

Suggest having 3 unconfirmed users, and assert that the creation order is different to the email address ordering